plzcrawler/plzCrawler.py

```diff
@@ -23,53 +23,63 @@ def __init__(self, excelTableNameString, workSheetString, tableDirectoryRawStrin
         self.placeIdListe = []
 
     def openAddressTable(self):
-        """nimmt Name einer Exceltabelle in Form eines Strings,
-        den Namen des entsprechenden Tabellenblattes in Form eines Strings,
-        & die Windows-Tabellen-Directory in Form eines Raw-Strings;
-        navigiert zum angegebenen Windows-Pfad & öffnet die Tabelle als pandas-dataframe;
-        speichert Tabelle als pandas dataframe-object"""
-
-        # zu directory navigieren, die Tabelle enthält
+        # """nimmt Name einer Exceltabelle in Form eines Strings,
+        # den Namen des entsprechenden Tabellenblattes in Form eines Strings,
+        # & die Windows-Tabellen-Directory in Form eines Raw-Strings;
+        # navigiert zum angegebenen Windows-Pfad & öffnet die Tabelle als pandas-dataframe;
+        # speichert Tabelle als pandas dataframe-object"""
+        '''
+        takes the name of an Excel spreadsheet in the form of a string,
+        the name of the corresponding spreadsheet in the form of a string,
+        & the Windows table directory in the form of a raw string;
+        navigate to the specified windows path & open the table as pandas-dataframe;
+        saves table as pandas dataframe-object
+        '''
+        # Navigate to directory containing table
         dir = os.chdir(self.tableDirectoryRawString)
         # print(Path.cwd())
 
         openedAddressTable = pd.read_excel(open(f'{self.excelTableNameString}.xlsx', 'rb'),
-                                          sheet_name=self.workSheetString)
+                                        sheet_name=self.workSheetString)
         # print(df)
         return openedAddressTable
 
     def getColumnLists(self, addressDataframe):
-        """nimmt pandas dataframe-object als input-argument;
-        holt relevante Spalten aus input-object heraus;
-        gibt ausgewählte Spaltenwerte in Listen gleicher Länge aus"""
+        '''
+        takes panda's dataframe object as an input argument;
+        pulls relevant columns out of input-object;
+        outputs selected column values in lists of equal length
+        '''
 
-        # Spalte "Namen" aus Input-Tabelle (dataframe-object) holen
+        # Get column "names" from input table (dataframe-object).

```

Comment:
please change "names" to "Namen" again, since it refers to the respective text in a spreadsheet column